easybuild/tools/module_generator.py

```diff
@@ -298,6 +299,100 @@ def use(self, paths, prefix=None, guarded=False):
         """
         raise NotImplementedError
 
+    def _generate_help_text(self):
+        """
+        Generate syntax-independent help text used for `module help`.
+        """
+
+        def generate_section(sec_name, sec_txt, strip=False):
+            """
+            Generate section with given name and contents.
+            """
+            if sec_txt:
+                if strip:
+                    text = sec_txt.strip()
+                else:
+                    text = sec_txt
+
+                return [
+                    '',
+                    '',
+                    sec_name,
+                    '=' * len(sec_name),
+                    text,
+                ]
+            else:
+                return []
```

Comment:
hmm, this is a bit too vertically verbose, no? also, we generally avoid inline `return`s

also, let's make this a private method like the others you're calling out to, rather than an inner function?

```python
def _generate_help_section(sec_name, sec_txt, strip=False):
    """
    Generate section with given name and contents.
    """
    res = []
    if sec_txt:
        if strip:
            sec_txt = sec_txt.strip()
        res = ['', '', sec_name, '=' * len(sec_name), sec_txt]
    return res
```